src/app/fdctl/src/commands/configure/frank.rs

```diff
@@ -56,31 +56,27 @@ fn step(config: &mut Config) {
     let mcache = run!("{bin}/fd_tango_ctl new-mcache {workspace} {} 0 0", config.tiles.pack.max_pending_transactions);
     
     let dcache = run!("{bin}/fd_tango_ctl new-dcache {workspace} 4808 {} 1 1 0", config.tiles.pack.max_pending_transactions);
-    let (pack_scratch, cu_est_table) = if Path::new(&format!("{bin}/fd_pack_ctl")).exists() {
-        // TODO: Always enable this when merged with pack changes in 1.2
-        let pack_scratch = run!("{bin}/fd_pack_ctl new-scratch {workspace} {} {}", config.tiles.pack.solana_labs_bank_thread_count, config.tiles.pack.max_pending_transactions);
-        let cu_est_table = run!("{bin}/fd_pack_ctl new-cu-est-tbl {workspace} {} {} {}", config.tiles.pack.compute_unit_estimator_table_size, config.tiles.pack.compute_unit_estimator_ema_history, config.tiles.pack.compute_unit_estimator_ema_default);
-        (pack_scratch, cu_est_table)
-    } else {
-        ("unused".to_string(), "unused".to_string())
-    };
 
     let return_fseq = run!("{bin}/fd_tango_ctl new-fseq {workspace} 0");
     run!("{bin}/fd_pod_ctl \
         insert {pod} cstr {prefix}.pack.cnc {cnc} \
         insert {pod} cstr {prefix}.pack.out-mcache {mcache} \
         insert {pod} cstr {prefix}.pack.out-dcache {dcache} \
-        insert {pod} cstr {prefix}.pack.scratch {pack_scratch} \
-        insert {pod} cstr {prefix}.pack.cu-est-tbl {cu_est_table} \
         insert {pod} cstr {prefix}.pack.return-fseq {return_fseq} \
         insert {pod} ulong {prefix}.pack.bank-cnt {solana_labs_bank_thread_count} \
         insert {pod} ulong {prefix}.pack.txnq-sz {max_pending_transactions} \
-        insert {pod} ulong {prefix}.pack.cu-est-tbl-sz {compute_unit_estimator_table_size} \
         insert {pod} uint {prefix}.pack.cu-limit {solana_labs_bank_thread_compute_units_executed_per_second}",
         solana_labs_bank_thread_count=config.tiles.pack.solana_labs_bank_thread_count,
         max_pending_transactions=config.tiles.pack.max_pending_transactions,
-        compute_unit_estimator_table_size=config.tiles.pack.compute_unit_estimator_table_size,
         solana_labs_bank_thread_compute_units_executed_per_second=config.tiles.pack.solana_labs_bank_thread_compute_units_executed_per_second);
+    
+    let cu_est_bin_cnt = 4096;
+    let cu_est_footprint = 32+32*cu_est_bin_cnt;
+    let cu_est_mem = run!("{bin}/fd_wksp_ctl alloc {workspace} 32 {}", cu_est_footprint);
+    run!("{bin}/fd_pod_ctl \
```

Comment:
Lame style nits but for Rust the formatting is a bit different. Names can be fully formed, eg `compute_unit_estimator_bucket_count` rather than `cu_est_bin_cnt`, and spaces between ops like `32 + 32 * cu_est_bin_cnt` (I think `cargo fmt` should catch this?)

Also can you merge these `fd_pod_ctl` into the prior one so there's only one call to the binary for pack?

Reply:
Thanks, it has been a long time since I've written Rust, so I appreciate the tips. I fixed these.